cw-relayer/tests/e2e/config/relayer-config.toml

```
@@ -8,7 +8,7 @@ max_tick_timeout = "1m"
 query_timeout = "5000ms"
 max_retries = 1
 
-gas_adjustment = 1.5
+gas = 1.5
```

Should probably update example config with this also if its required to be this much higher
```suggestion
gas = 15
```